modules/ROOT/pages/advanced-templates.adoc

```diff
@@ -0,0 +1,84 @@
+= Advanced Template plugin
+:navtitle: Advanced Template
+:description: Create and edit complex templates in {productname} for use with {productname}.
+:description_short: Create templates.
+:keywords: plugin, advanced template, templates, advtemplate.
+:pluginname: Advanced Template
+:plugincode: advtemplate
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.4v.adoc[]
+
+The Advanced Template Premium plugin enables the creation of complex, interactive templates using {productname}.
+
+Templates are declared as `const` variables that can be setup as part of a {productname} instance.
```

Comment:
```suggestion
Templates are declared as `const` variables that can be setup as part of a {productname} instance.
```
This needs to be removed since we aren't documenting the option any more

Reply:
Removed.